src/lib.rs

```diff
@@ -272,6 +276,34 @@ impl StreamDeck {
         self.write_button_image(key, &image)
     }
 
+    /// Sets a button to the provided text.
+    /// Will break text over \n linebreaks
+    pub fn set_button_text(
+        &mut self,
+        key: u8,
+        font: &Font,
+        pos: &TextPosition,
+        text: &str,
+        opts: &TextOptions,
```

Comment:
This is where I ended up with for defaults:
* I wasn't sure if we wanted to ship a font (Roboto?) with this library, so I left it as a required field. I figure users will have SOME font they like on their machine.
* I created a default size, and line height I figured look decent
* White text on a black background is always a good starting point I figured.

Rearranged the ordering a little too.

WDYT?

Reply:
seems good to me! good point on the font, defaults sound good, and if positioning wasn't critical i'd probably put it in the TextOptions but since it is, might as well stay out.

it's not critical / required here but it would be excellent to have an example using this (pulling a font from somewhere? idk) if you were interested in another PR.